docs/source/cql.rst

```diff
@@ -16,17 +21,20 @@ following predicates:
 Formats
 -------
 
-At the moment pygeoapi supports only the CQL dialect with the JSON encoding `CQL-JSON <https://portal.ogc.org/files/96288#simple-cql-JSON>`_.
+At the moment Elasticsearch supports only the CQL dialect with the JSON encoding `CQL-JSON <https://portal.ogc.org/files/96288#simple-cql-JSON>`_.
 
-Providers
----------
-
-As of now the available providers supported for CQL filtering are limited to only :ref:`Elasticsearch <Elasticsearch>`.
+PostgreSQL supports both CQL-JSON and CQL-text dialects, `CQL-JSON <https://portal.ogc.org/files/96288#simple-cql-JSON>`_ & `CQL-TEXT <https://portal.ogc.org/files/96288#simple-cql-text>`_
 
 Queries
 ^^^^^^^
 
-The following type of queries are supported right now:
+The PostgreSQL provider uses `pygeofilter <https://github.com/geopython/pygeofilter>`_ allowing a range of filter expressions, see examples for:
+
+* `Comparison predicates <https://portal.ogc.org/files/96288#simple-cql_comparison-predicates>`_
+* `Spatial predicates <https://portal.ogc.org/files/96288#enhanced-spatial-operators>`_
+* `Temporal predicates <https://portal.ogc.org/files/96288#simple-cql_temporal>
+
+Using ElasticSearch the following type of queries are supported right now:
```

Comment:
`s/ElasticSearch/Elasticsearch/g`

Reply:
Sorry I don't understand that syntax, what do you mean?

Reply:
sorry for my sed commands :)  Replace `ElasticSearch` with `Elasticsearch`, globally.

Reply:
Sorry bit of googling & I got there!